packages/frontend/core/src/layouts/workspace-layout.tsx

```diff
@@ -41,6 +41,7 @@ import { useRegisterFindInPageCommands } from '../hooks/affine/use-register-find
 import { useNavigateHelper } from '../hooks/use-navigate-helper';
 import { useRegisterWorkspaceCommands } from '../hooks/use-register-workspace-commands';
 import { QuickSearchService } from '../modules/cmdk';
+import { CMDKQuickSearchModal } from '../modules/cmdk/views';
```

Comment:
I think CMDKQuickSearchModal without Suspense caused this issue. Maybe adding a suspense to still allowing chunk splitting & loading instead of making all of them bundled?

Reply:
I measured their bundle sizes and they barely changed.